test/unit/api/autorequire-pouchdb-replication-test.js

```diff
@@ -0,0 +1,26 @@
+var test = require('tap').test
+var proxyquire = require('proxyquire').noPreserveCache()
+var state = {}
+
+state.PouchDB = function () {
+  return {}
+}
+
+state.PouchDB.plugin = function () {}
+
+test('Store replicate function', function (group) {
+  group.test('Returns a promise rejection when the module is not found', function (t) {
+    var replicate = proxyquire('../../../api/store/replicate', {'pouchdb-replication': null})
```

Comment:
can you please change this to 

```js
var replicate = proxyquire('../../../api/store/replicate', {
  'pouchdb-replication': null
})
```

we try to keep lines short if possible for better readability